### PR TITLE
Fixes #10705: filter out non-installable Systems BZ1228292.

### PR DESCRIPTION
**Pull request by waldenraines**
http://projects.theforeman.org/issues/107050
https://bugzilla.redhat.com/show_bug.cgi?id=1228292

### CODE REVIEW

**Comment by waldenraines** [comment]
Note to self: @ehelms wants a test (and rightfully so)

**Comment by waldenraines** [comment]
@ehelms updated.

File: test/models/system_test.rb [hunk 1]
```diff
@@ -168,6 +168,10 @@ def test_with_installable_errata
 
       systems = System.with_installable_errata([non_installable.first])
       refute systems.include?(@errata_system)
+
+      systems = System.with_installable_errata([installable.first, non_installable.first])
+      assert_include systems, @errata_system
+      assert_equal systems.length, 1
```
Comment (ehelms) [inline test/models/system_test.rb]: Based on what this intended to fix, I think an assertion showing that a system that has errata from `installable` does not get returned when that is combined with the `non_installable` errata (that would in theory, also be applicable to the system).

**Comment by waldenraines** [comment]
@ehelms updated.

**Comment by ehelms** [comment]
ACK

**Pull request merged**